source/simulator/assembly.cc

```diff
@@ -525,6 +525,7 @@ namespace aspect
 
         Amg_data.smoother_sweeps = 2;
         Amg_data.coarse_type = "symmetric Gauss-Seidel";
+        Amg_data.constant_modes=constant_modes;
```

Comment:
Sorry to nitpick, but just to keep our code style here consistent, please make this into:
```suggestion
        Amg_data.constant_modes = constant_modes;
```